pkg/util/kubernetes/pod.go

```diff
@@ -50,24 +50,33 @@ func natsPodContainer(clusterName, version string, serverImage string) v1.Contai
 		},
 		Name:  constants.NatsContainerName,
 		Image: MakeNATSImage(version, serverImage),
-		Ports: []v1.ContainerPort{
-			{
-				Name:          "cluster",
-				ContainerPort: int32(constants.ClusterPort),
-				Protocol:      v1.ProtocolTCP,
-			},
-			{
-				Name:          "client",
-				ContainerPort: int32(constants.ClientPort),
-				Protocol:      v1.ProtocolTCP,
-			},
-			{
-				Name:          "monitoring",
-				ContainerPort: int32(constants.MonitoringPort),
-				Protocol:      v1.ProtocolTCP,
-			},
+	}
+
+	ports := []v1.ContainerPort{
```

Comment:
This change is for the case no monitoring or routes are setup, right?

Reply:
yes those are the defaults for cluster and monitoring ports, pnly change is for the client port to be able to bind the host port below that can be used to connect to it using the host network: `port.HostPort = int32(constants.ClientPort)`